src/rpdk/core/data/managed-upload-infrastructure.yaml

```diff
@@ -117,13 +117,19 @@ Resources:
           Statement:
           - Effect: Allow
             Action:
+            - kms:Encrypt
+            - kms:Decrypt
+            - kms:ReEncrypt*
+            - kms:GenerateDataKey*
+            - kms:DescribeKey
             - logs:CreateLogGroup
             - logs:CreateLogStream
             - logs:DescribeLogGroups
             - logs:DescribeLogStreams
             - logs:PutLogEvents
             - cloudwatch:ListMetrics
             - cloudwatch:PutMetricData
+            - s3:PutObject
```

Comment:
This seems particularly dangerous to leave wide open. Can we scope it to just the bucket above?

Reply:
https://github.com/aws-cloudformation/cloudformation-cli/issues/466